File: ReleaseNotesGenerator/ClientApp/app/common/classes/ReleaseNote.ts

    import { Repository } from './Repository';
import { EntityBase } from './EntityBase';
    export class ReleaseNote extends EntityBase<number> {
        
        notes: string;
        created: Date;
        repositoryId: number;
        repository: Repository;
        repositoryPath: string;
        branchName: string;
        from: Date;
        until: Date;
        }

    

File: ReleaseNotesGenerator/Features/ReleaseNotes/ReleaseNote.cs
using System;
using ReleaseNotesGenerator.Features.SourceCodeRepositories;

namespace ReleaseNotesGenerator.Features.ReleaseNotes
{
    public class ReleaseNote : EntityBase<int>
    {
        public string Notes { get; set; }

        public DateTime Created { get; set; }

        public int RepositoryId { get; set; }

        public Repository Repository { get; set; }

        public string RepositoryPath { get; set; }

        public string BranchName { get; set; }

        public DateTime From { get; set; }

        public DateTime Until { get; set; }
    }
}